docs/topics/geolocalized_entities.md

```diff
@@ -0,0 +1,120 @@
+# Entities geolocalization
+
+The platform allows entities geolocalization. In other words, entities may have an attribute which 
+value is interpreted as the entity location, among the following types:
+
+- A point, e.g. to model a Car entity which moves across a city.
+- A line, e.g. to model a Street entity
+- A polygon with an arbitrary number of vertex, e.g. to model a city neighborhood
+- An arbitrary [GeoJSON](http://geojson.org)
+
+Entities location is managed through the [Data API](../data_api.md) exposed by [Context Broker](../context_broker.md). See details at "Geospacial properties of entities" section at the [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable/). The following example
+shows how to set a `location` attribute for a given entity (which id is `Tenerife`) and which roughly corresponds to the Tenerife island surface
+on Earth:
+
+    PATCH /v2/entities/Tenerife/attrs/location
+    Content-Type: application/json
+
+    {
+      "value": [ "28.362, -16.883", "28.003, -16.693", "28.552, -16.136", "28.520, -16.418", "28.396, -16.641", "28.362, -16.883" ]
+      "type": "geo:polygon"
+    }
+
+
+With regards with entities corresponding to devices connected to the platform and managed by [IoT Agents](../device_gateway.md), you can
+geo-locate them as `geo:point` using the model transformation functionality. Let's assume the same scenario shown in the [notifications scenario](how_notifications_work.md).
+Sensor information can be reported either directly in the `latitude, longitude` format, or separately, by using expressions to combine the information
+into a geo point. To do so, the car provisioning should be changed to reflect this new information. If the sensor information is sent using :
+
+    POST /iot/services
+    Content-Type: application/json
+    Fiware-service: smartown
+    Fiware-servicepath: /roads
+
+    {
+      "services": [
+        {
+          "protocol": [
+                  "IoTA-UL"
+                ],
+          "apikey": "801230BJKL23Y9090DSFL123HJK09H324HV8732",
+          "entity_type": "Car",
+          "attributes": [
+            {
+              "object_id": "s",
+              "name": "speed",
+              "type": "Boolean"
+            },
+            {
+              "object_id": "la",
+              "name": "latitude",
+              "type": "Integer"
+            },
+            {
+              "object_id": "lo",
+              "name": "longitude",
+              "type": "Integer"
+            },
+            {
+              "name":"location",
```

Comment:
Now it looks like:

    "name" : "location",

but it should be

    "name": "location",